### PR TITLE
Update e2e go.mod

### PR DESCRIPTION
**Pull request by liangping**
Fix: compiling error.
replace IBC module to the local, which is convenient for development

## Description

closes: #XXXX

---

Before we can merge this PR, please make sure that all the following items have been
checked off. If any of the checklist items are not applicable, please leave them but
write a little note why.

- [ ] Targeted PR against correct branch (see [CONTRIBUTING.md](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#pr-targeting))
- [ ] Linked to Github issue with discussion and accepted design OR link to spec that describes this work.
- [ ] Code follows the [module structure standards](https://github.com/cosmos/cosmos-sdk/blob/master/docs/building-modules/structure.md).
- [ ] Wrote unit and integration [tests](https://github.com/cosmos/ibc-go/blob/master/CONTRIBUTING.md#testing)
- [ ] Updated relevant documentation (`docs/`) or specification (`x/<module>/spec/`)
- [ ] Added relevant `godoc` [comments](https://blog.golang.org/godoc-documenting-go-code).
- [ ] Added a relevant changelog entry to the `Unreleased` section in `CHANGELOG.md`
- [ ] Re-reviewed `Files changed` in the Github PR explorer
- [ ] Review `Codecov Report` in the comment section below once CI passes

### CODE REVIEW

File: e2e/go.mod [hunk 1]
```diff
@@ -169,4 +169,8 @@ require (
 )
 
 // TODO: using version v1.0.0 causes a build failure. This is the previous version which compiles successfully.
-replace github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
+replace (
+	github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
+	github.com/ChainSafe/go-schnorrkel/1 => github.com/ChainSafe/go-schnorrkel v1.0.0
+	github.com/cosmos/ibc-go/v5 => ../
```
Comment (chatton) [inline e2e/go.mod:175]: I'm a bit hesitant to pin this.

Currently, `ibctest` is pinned to v5, if we update to v6, the tests will fail due to a duplicate message registration with multiple `ibc-go` imports.

We're explicitly keeping it separate right now, but I think having this as a commented line at the bottom of the file would be very useful for development, i.e.

```
// uncomment to use the local version of ibc-go
// replace github.com/cosmos/ibc-go/v5 => ../`
```

Comment (liangping) [reply e2e/go.mod:175]: OK, Yeah, it's better.

**Comment by chatton** [comment]
Hi @liangping, it looks like you'll need to run `go mod tidy` also.

**Comment by liangping** [comment]
I did.

**Comment by liangping** [comment]
Let me add it to the comment

**Review by colin-axner — APPROVED** [review]

**Review by chatton — APPROVED** [review]
LGTM thanks @liangping

**Pull request merged**